src/zimscraperlib/imaging.py

```diff
@@ -40,8 +40,27 @@ def solarize(r, g, b):
     return rgb_to_hex(mr, mg, mb), rgb_to_hex(sr, sg, sb)
 
 
+def alpha_not_supported():
+    """ list of PIL image formats which do not support alpha layer """
+    return ["JPEG", "BMP", "EPS", "PCX"]
+
+
+def save_image(image, dst, fmt, params=None):
+    """ saves an image with given format and default args and overrides them if params are given 
+        params: PIL params as dict (https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html) """
+    default_args = {"JPEG": {"quality": 100}, "PNG": {}}
+    args = params if params else default_args.get(fmt, {})
```

Comment:
I think we want our default args always but `update()` them with what was passed.
Should we use keyword arguments instead of a params dict?

Reply:
Done

Reply:
Can you push it then?